src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1393,11 +1398,16 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     // - location: the location of the mouse during this event. This location is
     //   relative to the origin of the control
     // - delta: the mouse wheel delta that triggered this event.
+    // - state: the state for each of the mouse buttons (pressed/unpressed)

```

Comment:
this comment isn't right, but I also don't care for us to comment three booleans that mean the same obvious thing ;P